### DISCUSSION TITLE
Prevent upscaling for small images

### DISCUSSION THREAD

**Discussion by dteskera** [directus/directus#10431] [Feature Requests]
is it possible to prevent upscaling of small images in image interface?

**Comment by benhaynes** [comment]
Interesting, I think it always scales them up to the interface size... but I agree that it _should_ use the native size as the max, and then center/center. Would love a PR for this!

**Comment by lukas-schaetzle** [comment]
I wanted to give it a shot but was unsure, how to proceed.

Currently the interface always takes 400px in height. Should this be left untouched and only center the image in the container vertically + horizontally (see 1st screenshot)? (Personally, I would shrink the whole interface height if it's too big, see 2nd screenshot, but maybe with a small padding on top and bottom?)

And should big images (native image size > max interface size) still cover the whole interface container? With my current simple CSS only adjustment big images would also be fully contained in the container and there are likely black bars even if the image's native resolution is higher so it could be scaled and cut in such a way that it covers the whole container (see 3rd screenshot). I think, to keep the current covering behaviour for only big enough images, some JS tinkering would be necessary... But then again, maybe its better for a CMS to show the full picture without cutting off some parts (which currently happens)?

![image34](https://user-images.githubusercontent.com/45203603/157489686-0c915814-9a0a-43b8-ba92-4285a0f3ecc2.png)

**Reply by benhaynes** [reply]
Yeah, I think we center smaller images and give them a little padding all around (we typically use 20/40px for things like this). For the cover/contain question, we already have an option in the interface to configure this. Feel free to confirm, but I think that's how this works.

**Reply by lukas-schaetzle** [reply]
Ah yes, I somehow missed that checkbox.
We should also make sure to prevent the scaling probelm in the card displays. As far as I can tell, they always use the cover type. Should we just switch here to the css scale-down option for them? With that it would like this for example:

![Unbenannt](https://user-images.githubusercontent.com/45203603/157790325-fbf7d1dd-5735-47a4-80f6-6ebb1ffc1505.PNG)

Or would you keep using covering images if they are big enough?

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
